nexus/src/app/instance.rs

```diff
@@ -175,6 +174,7 @@ impl super::Nexus {
         //
         // TODO Even worse, post-authz, we do two lookups here instead of one.
         // Maybe sagas should be able to emit `authz::Instance`-type objects.
+
```

Comment:
Nit: Was this newline on purpose?

Reply:
I don't remember.  I just removed it in 33951e28e39cfd0dfeacf5066700dff6f206b045.